google/cloud/storage/tests/testbench.py

```diff
@@ -569,6 +569,34 @@ def get_revision_by_generation(self, generation):
     def get_latest(self):
         return self.revisions.get(self.generation, None)
 
+    def check_preconditions_by_value(
+            self, generation_match, generation_not_match, metageneration_match,
+            metageneration_not_match):
+        """Verify that the given precondition values are met."""
+        if generation_match is not None \
```

Comment:
Prefer enclosing in parentheses over backslashed-linebreaks for conditionals that span multiple lines.

    if (generation_match is not None
            and int(generation_match) != self.generation):

Reply:
Fixed, here and elsewhere in the file.